get.php

```diff
@@ -124,6 +124,7 @@
 	"scrsw" => "https://github.com/javidominguez/screenshots/releases/download/2.0/screenshots-2.0.nvda-addon",
 	"searchwith" => "https://github.com/ibrahim-s/searchWith/releases/download/v2.2/searchWith-2.2.nvda-addon",
 	"sentencenav" => "https://github.com/mltony/nvda-sentence-nav/releases/download/v2.12/SentenceNav-2.12.nvda-addon",
+	"skype" => "https://www.dlee.org/skype/NVDA/skype.nvda-addon",
```

Comment:
I think this is fine for community add-ons website, but keep in mind that a version tag must be provided as part of the name to help people and update tools differentiate versions.